pepper-apis/studybuilder-cli/studies/singular/patches/ddp-8554-self-validation.conf

```diff
@@ -0,0 +1,149 @@
+{
+  "validations": [
+    {
+      "messageTemplate": {
+        "templateType": "HTML",
+        "templateText": "$singular_consent_self_validation",
+        "variables": [
+          {
+            "name": "singular_consent_self_validation",
+            "translations": [
+              {
+                "language": "en",
+                "text": """Please check the date of birth you have entered. If the date is correct, then you
+                               are younger than the age required by local law to self-enroll in this study. However,
+                               you may ask your parent or guardian to enroll you in the study. If you think there
+                               is a mistake, please reach out to us at (650) 561-6750 or
+                               <a href="mailto:contact@projectsingular.org">contact@projectsingular.org</a>.
+                            """
+              }
+            ]
+          }
+        ]
+      },
+      "stableIds": ["PREQUAL_AGE", "CONSENT_SELF_DATE_OF_BIRTH", "PREQUAL_COUNTRY", "PREQUAL_STATE", "PREQUAL_PROVINCE"],
+      "precondition": """
+                          user.studies["singular"].forms["PREQUAL"].questions["PREQUAL_AGE"].isAnswered()
+                          &&
+                          user.studies["singular"].forms["CONSENT_SELF"].questions["CONSENT_SELF_DATE_OF_BIRTH"].isAnswered()
+                          &&
+                          user.studies["singular"].forms["PREQUAL"].questions["PREQUAL_COUNTRY"].isAnswered()
```

Comment:
prequal is NOT even associated to the child being enrolled and hence NONE of these validations will even trigger I believe.
user.studies["singular"].forms["PREQUAL"] is looking for prequal associated with child .
prequal is associated with ADULT (proxy)not the child.
Need to check and test.

Reply:
This validation is not for the child. This one is for self-enrollment.

Reply:
Looks like I added it here when I meant to add to child-enrollment